2024-12-04 03:04:00,667 - ERROR - data_validation_check.py  - ["Outlier detected in column 'Wet Bulb Temperature': 11.5 is out of range (11.799999999999997, 12.800000000000002)", "Outlier detected in column 'Maximum Wind Speed': 2.3 is out of range (2.3999999999999986, 7.900000000000001)", "Outlier detected in column 'Wet Bulb Temperature': 13.0 is out of range (11.799999999999997, 12.800000000000002)", "Outlier detected in column 'Maximum Wind Speed': 9.2 is out of range (2.3999999999999986, 7.900000000000001)"]
2024-12-04 03:04:00,703 - ERROR - data_validation_check.py  - Failed records saved to: quarantine\20241204_030400_failed_records.csv
2024-12-04 03:07:24,167 - ERROR - data_validation_check.py  - ["Outlier detected in column 'Wet Bulb Temperature': 11.5 is out of range (11.799999999999997, 12.800000000000002)", "Outlier detected in column 'Maximum Wind Speed': 2.3 is out of range (2.3999999999999986, 7.900000000000001)", "Outlier detected in column 'Wet Bulb Temperature': 13.0 is out of range (11.799999999999997, 12.800000000000002)", "Outlier detected in column 'Maximum Wind Speed': 9.2 is out of range (2.3999999999999986, 7.900000000000001)"]
2024-12-04 03:07:24,197 - ERROR - data_validation_check.py  - Failed records saved to: quarantine\20241204_030724_failed_records.csv
2024-12-04 03:10:44,132 - ERROR - data_validation_check.py  - ["Outlier detected in column 'Solar Radiation': 588 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 658 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 502 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 287 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 168 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 122 is out of range (-62.0, 93.0)", "Outlier detected in column 'Wind Direction': 348 is out of range (-74.75, 280.25)", "Outlier detected in column 'Barometric Pressure': 1002.4 is out of range (997.25, 1002.3749999999999)", "Outlier detected in column 'Barometric Pressure': 1002.4 is out of range (997.25, 1002.3749999999999)", "Outlier detected in column 'Wet Bulb Temperature': 11.9 is out of range (15.8, 21.799999999999997)", "Outlier detected in column 'Interval Rain': 0.1 is out of range (0.0, 0.0)", "Outlier detected in column 'Wet Bulb Temperature': 11.9 is out of range (15.8, 21.799999999999997)", "Outlier detected in column 'Wind Speed': 5.3 is out of range (-1.6249999999999991, 5.124999999999999)", "Outlier detected in column 'Solar Radiation': 293 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 136 is out of range (-62.0, 93.0)", "Outlier detected in column 'Wind Direction': 356 is out of range (-74.75, 280.25)", "Outlier detected in column 'Solar Radiation': 143 is out of range (-62.0, 93.0)", "Outlier detected in column 'Wet Bulb Temperature': 12.9 is out of range (15.8, 21.799999999999997)"]
2024-12-04 03:10:44,163 - ERROR - data_validation_check.py  - Failed records saved to: quarantine\20241204_031044_failed_records.csv
2024-12-04 03:10:46,903 - ERROR - data_validation_check.py  - ["Outlier detected in column 'Solar Radiation': 588 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 658 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 502 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 287 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 168 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 122 is out of range (-62.0, 93.0)", "Outlier detected in column 'Wind Direction': 348 is out of range (-74.75, 280.25)", "Outlier detected in column 'Barometric Pressure': 1002.4 is out of range (997.25, 1002.3749999999999)", "Outlier detected in column 'Barometric Pressure': 1002.4 is out of range (997.25, 1002.3749999999999)", "Outlier detected in column 'Wet Bulb Temperature': 11.9 is out of range (15.8, 21.799999999999997)", "Outlier detected in column 'Interval Rain': 0.1 is out of range (0.0, 0.0)", "Outlier detected in column 'Wet Bulb Temperature': 11.9 is out of range (15.8, 21.799999999999997)", "Outlier detected in column 'Wind Speed': 5.3 is out of range (-1.6249999999999991, 5.124999999999999)", "Outlier detected in column 'Solar Radiation': 293 is out of range (-62.0, 93.0)", "Outlier detected in column 'Solar Radiation': 136 is out of range (-62.0, 93.0)", "Outlier detected in column 'Wind Direction': 356 is out of range (-74.75, 280.25)", "Outlier detected in column 'Solar Radiation': 143 is out of range (-62.0, 93.0)", "Outlier detected in column 'Wet Bulb Temperature': 12.9 is out of range (15.8, 21.799999999999997)"]
2024-12-04 03:10:46,983 - ERROR - data_validation_check.py  - Failed records saved to: quarantine\20241204_031046_failed_records.csv
